### ISSUE TITLE
E444: Cannot close last window

### ISSUE THREAD

**Issue by YodaEmbedding** [dstein64/nvim-scrollview#10]
**Steps to reproduce**

Minimal .vimrc:

```vim
" ~/vimrc_minimal
call plug#begin('~/.local/share/nvim/plugged')
Plug 'dstein64/nvim-scrollview'
call plug#end()
```

Run the following:

```bash
mkdir -p root/subdir
cd root
touch a subdir/b
vim -u ~/vimrc_minimal
```

Inside vim, execute the following:

```vim
:e a<CR>
:e subdir/b<CR>
200OHello, world!<Esc>
:w<CR>
:bd<CR>
```

We should now see the error message:

```vim
E444: Cannot close last window
E444: Cannot close last window
Press ENTER or type command to continue
```

**Comment by dstein64** [comment]
Thanks for reporting this @YodaEmbedding!

I've reproduced the issue outside the context of `nvim-scrollview` and reported the issue to the Neovim project.
https://github.com/neovim/neovim/issues/13628

**Comment by dstein64** [comment]
This problem is now documented under `:help scrollview-issues`, along with two proposed workarounds:

- Workaround 1: Use `:only` or `<ctrl-w>o`, which has the side-effect of closing all floating windows (including scrollbars). A subsequent `:bdelete` command works without error.
- Workaround 2: Create a `Bdelete` command that avoids the issue.
```
  command Bdelete
        \ silent! ScrollViewDisable
        \ | bdelete
        \ | silent! ScrollViewEnable
```
(workaround 2 requires a plugin update)

**Comment by dstein64** [comment]
This is fixed as of Neovim [#17806](https://github.com/neovim/neovim/pull/17806).
